resources/views/inputs/markdown.blade.php

```diff
@@ -91,3 +92,8 @@ class="overflow-hidden bg-white rounded border-2 border-theme-secondary-200"
     </div>
 </div>
 
+@push($scriptsStackName)
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.4/codemirror.min.css" />
+<script src="https://uicdn.toast.com/editor/latest/toastui-editor-all.min.js" defer></script>
+<link rel="stylesheet" href="https://uicdn.toast.com/editor/latest/toastui-editor-only.min.css" />
+@endpush
```

Comment:
better to have the parent that calls the component handle this, so it's possible to have multiple markdown editors on a single page for example without duplicate js/css includes. Also saves on needing a property for the `stack` call